tests/chart_tests/test_fluentd_daemonsets.py

```diff
@@ -0,0 +1,28 @@
+from tests.chart_tests.helm_template_generator import render_chart
+import pytest
+from tests import supported_k8s_versions, get_containers_by_name
+
+
+@pytest.mark.parametrize(
+    "kube_version",
+    supported_k8s_versions,
+)
+class TestFluentdConfig:
+    show_only = ["charts/fluentd/templates/fluentd-daemonset.yaml"]
+
+    def test_fluentd_daemonset(self, kube_version):
+        docs = render_chart(
+            kube_version=kube_version,
+            show_only=self.show_only,
+        )
+        # assert len(docs) == 1
+        doc = docs[0]
+        c_by_name = get_containers_by_name(doc)
+        # print (c_by_name)
+        assert c_by_name["fluentd"]["name"] == "fluentd"
+        liveness_probe = c_by_name["fluentd"]["livenessProbe"]
+        assert liveness_probe["failureThreshold"] == 3
+        assert liveness_probe["initialDelaySeconds"] == 30
+        assert liveness_probe["periodSeconds"] == 15
+        assert liveness_probe["successThreshold"] == 1
```

Comment:
there is already a file called test_fluentd.py you can add tests there new file not required

Reply:
@Shubham-astro I agree with @pgvishnuram. We can move these assertions inside of this class https://github.com/astronomer/astronomer/blob/1c9dde172c/tests/chart_tests/test_fluentd.py#L12

Also we should rename this function to be more accurate, because what it is testing is not described in the test name: https://github.com/astronomer/astronomer/blob/1c9dde172c/tests/chart_tests/test_fluentd.py#L20

Reply:
done